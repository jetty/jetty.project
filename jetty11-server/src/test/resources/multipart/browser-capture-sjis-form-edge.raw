-----------------------------7e28636151054
Content-Disposition: form-data; name="japanese"

健治
-----------------------------7e28636151054
Content-Disposition: form-data; name="hello"

ャユ戆タ
-----------------------------7e28636151054--
