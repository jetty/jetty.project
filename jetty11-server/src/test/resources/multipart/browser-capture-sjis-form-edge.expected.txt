Request-Header|Accept|text/html, application/xhtml+xml, image/jxr, */*
Request-Header|Accept-Encoding|gzip, deflate
Request-Header|Accept-Language|en-US
Request-Header|Cache-Control|no-cache
Request-Header|Connection|keep-alive
Request-Header|Content-Length|255
Request-Header|Content-Type|multipart/form-data; boundary=---------------------------7e28636151054
Request-Header|Host|localhost:9090
Request-Header|Referer|http://localhost:9090/sjis-form.html
Request-Header|User-Agent|Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36 Edge/16.16299
Parts-Count|2
Part-ContainsContents|japanese|健治
Part-ContainsContents|hello|ャユ戆タ